shared/reducers/chat2.tsx

```diff
@@ -941,6 +941,28 @@ export default (_state: Types.State = initialState, action: Actions): Types.Stat
       case EngineGen.chat1ChatUiChatInboxLayout: {
         try {
           const layout: RPCChatTypes.UIInboxLayout = JSON.parse(action.payload.params.layout)
+          if (!draftState.inboxHasLoaded) {
+            // on first layout, initialize any drafts and muted status
```

Comment:
this seems tricky to me. maybe we should remove it from the meta object so the gui doesnt' reference it and the saga that emits this can just have it as an additional payload? aka in `metasReceived` that has `action.payload.draftMap` etc

Reply:
To make sure I understand, the steps would be:

1.) Remove `isMuted` and `draft` from `Types.ConversationMeta`
2.) In the saga that dispatches `metasReceived`, we dispatch a new action to set muted and drafts.